stl/inc/ranges

```diff
@@ -776,6 +777,72 @@ namespace ranges {
         };
     };
 
+    template <class _Fn, class... _Types>
+    class _Range_closure : public _Pipe::_Base<_Range_closure<_Fn, _Types...>> {
+    public:
+        // We assume that _Fn is the type of a customization-point object. That means
+        // 1. The behavior of operator() is independent of cvref qualifers, so we can  use `invocable<_Fn, ` without

```

Comment:
Typo: "qualifers" should be "qualifiers". Also, there are two spaces between `can  use`.